linux_os/guide/system/network/network-ipv6/disabling_ipv6/grub2_ipv6_disable_argument/tests/cmd_line_commented_etcdefaultgrub.fail.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bas
+# platform = Red Hat Enterprise Linux 7,sle12,sle15
```

Comment:
You can't use the short IDs, you should use full names:

```
# platform = Red Hat Enterprise Linux 7,Oracle Linux 7,SUSE Linux Enterprise 12,SUSE Linux Enterprise 15
```

Reply:
used a multi_platform_sle instead , anyways that was the idea